pkgs/top-level/all-packages.nix

```diff
@@ -5775,6 +5775,8 @@ with pkgs;
 
   jellyseerr = callPackage ../servers/jellyseerr { };
 
+  overseerr = callPackage ../servers/overseerr { };
```

Comment:
I believe it'd be more appropriate to place the package in `pkgs/by-name/...` instead of listing it here as there are no overrides being done here.